project/plugins.sbt

```diff
@@ -1,5 +1,8 @@
+resolvers += Resolver.sonatypeRepo("releases")
+
 addSbtPlugin("org.scala-js"      % "sbt-scalajs" % "0.6.25")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"  % "5.0.0")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.9")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
-addSbtPlugin("org.scalastyle"    %% "scalastyle-sbt-plugin" % "1.0.0")
\ No newline at end of file
+addSbtPlugin("org.scalastyle"    %% "scalastyle-sbt-plugin" % "1.0.0")
```

Comment:
why is this getting added?

Reply:
It was still there. I just upgraded the version from 0.8 to 1.0 in my previous PR.